.gitignore

```diff
@@ -18,4 +19,8 @@ test/depth/subset-paths/*.out
 
 *.json
 og_to_gfa.py
-compute_maxes.py
\ No newline at end of file
+compute_maxes.py
+
+pollen/.DS_Store
```

Comment:
FWIW, the plain `.DS_Store` line above will cover this.

Reply:
Thank you! I've removed that line from the `.gitignore` to clean it up a bit